lib/plugins/aws/provider/awsProvider.js

```diff
@@ -140,6 +140,66 @@ class AwsProvider {
 
       // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8016
       serverless.configSchemaHandler.defineProvider('aws', {
+        provider: {
+          properties: {
+            httpApi: {
+              type: 'object',
+              properties: {
+                authorizers: {
+                  type: 'object',
+                  additionalProperties: {
+                    type: 'object',
+                    properties: {
+                      identitySource: { type: 'string' },
+                      issuerUrl: { type: 'string' },
+                      audience: { type: 'array', items: { type: 'string' } },
+                    },
+                    required: ['identitySource', 'issuerUrl', 'audience'],
+                    additionalProperties: false,
+                  },
+                },
+                cors: {
+                  oneOf: [
+                    { type: 'boolean' },
+                    {
+                      type: 'object',
+                      properties: {
+                        allowedOrigins: { type: 'array', items: { type: 'string' } },
+                        allowedHeaders: { type: 'array', items: { type: 'string' } },
+                        allowedMethods: { type: 'array', items: { type: 'string' } },
+                        allowCredentials: { type: 'boolean' },
+                        exposedResponseHeaders: { type: 'array', items: { type: 'string' } },
+                        maxAge: { type: 'integer', minimum: 0 },
+                      },
+                      additionalProperties: false,
+                    },
+                  ],
+                },
+                id: { type: 'string' },
+                payload: { type: 'string' },
+              },
+              additionalProperties: false,
+            },
+            logs: {
+              type: 'object',
+              properties: {
+                httpApi: {
+                  oneOf: [
+                    { type: 'boolean' },
+                    {
+                      type: 'object',
+                      properties: {
+                        format: { type: 'string' },
+                      },
+                      required: ['format'],
```

Comment:
I would not make it `required`, as technically it isn't.

ofc it doesn't seem logical to make object configuration without providing a `format` but still we should support configurations as `logs.httpApi = {}`